lib/route-helpers.js

```diff
@@ -49,7 +49,8 @@ function toCashOutTx (row) {
 function fetchPhoneTx (phone) {
   const sql = `select * from cash_out_txs
     where phone=$1 and dispense=$2
-    and (extract(epoch from (coalesce(confirmed_at, now()) - created))) * 1000 < $3`
+    and (extract(epoch from (now() - created))) * 1000 < $3
+    and confirmed_at is not null`
```

Comment:
Looks like you found the bug, but I don't think it should be testing `confirmed_at is not null` because I think we need to pull these records even before there's a confirmation.